src/components/TabNavigation/TabNavigation.test.tsx

```diff
@@ -33,7 +41,7 @@ describe('The TabNavigation component', () => {
     expect(tabs[1].getAttribute('aria-selected')).toEqual('false');
   });
 
-  it('shows selected tab in the foreground', () => {
+  xit('shows selected tab in the foreground', () => {
```

Comment:
maybe a `setTimeout` after the `userEvent` call will help here? (to give the DOM time to update.)

Reply:
I will give it a try! I completely forgot about this test, I thought only the Apollo-related ones were being skipped.